src/TeleportJoin.sol

```diff
@@ -211,6 +237,20 @@ contract TeleportJoin {
         emit Mint(hashGUID, teleportGUID, amtToTake, maxFeePercentage, operatorFee, msg.sender);
     }
 
+    /**
+    * @dev External authed function that registers the teleport
+    * @param teleportGUID Struct which contains the whole teleport data
+    **/
+    function registerMint(
+        TeleportGUID calldata teleportGUID
+    ) external auth {
+        bytes32 hashGUID = getGUIDHash(teleportGUID);
+        require(!teleports[hashGUID].blessed, "TeleportJoin/already-blessed");
+        teleports[hashGUID].blessed = true;
+        teleports[hashGUID].pending = teleportGUID.amount;
+        emit Register(hashGUID, teleportGUID);
+    }
+
```

Comment:
for `requestMint`: instead of `auth` it would be more explicit to just `require(msg.sender == oracleAuth)` (where `oracleAuth` is either defined in the constructor or `file`d)

Reply:
I would like to keep this open. We may add additional minting mechanisms later. IE maybe we want to add an optimistic bridge minter in the future.